nni/compression/pytorch/utils/mask_conflict.py

```diff
@@ -249,6 +252,10 @@ def fix_mask(self):
                         else:
                             channel_masks.append(
                                 (mask.abs().sum(1) != 0).int())
+                    elif type(m).__name__ == 'Embedding':
+                        if self.conv_prune_dim == 0:
+                            channel_masks.append(
+                                (mask.abs().sum(0) != 0).int())
```

Comment:
a little confused, when we prune embedding layer, the dimensions of weight are [vocab_size, embedding_dim], how to prune embedding layer at dimension 0?  I guess that when conv_prune_dim == 0,  the output_features of embedding layer will be pruned?

Reply:
yes, `conv_prune_dim == 0` means prune dimension is output feature